source/openapi-admin-v3.yaml

```diff
@@ -5132,55 +5178,517 @@ components:
         - name
         - function_source
         - respond_result
-    UserProvider:
-      type: object
-      properties:
-        id:
-          type: string
-        provider_type:
-          "$ref": "#/components/schemas/ProviderType"
-        provider_id:
-          type: string
-    ProviderSummary:
+    
+
+
+
+
+
+
+
+
+
+    BaseAuthProvider:
+      required:
+        - name
+        - type
       properties:
         _id:
           type: string
+          description: The authentication provider's unique ID.
         name:
           type: string
           description: The name of the authentication provider. This value must be the same as the value of `type`.
         type:
           "$ref": "#/components/schemas/ProviderType"
         disabled:
           type: boolean
-    FullProvider:
+          description: If `true`, the auth provider is disabled. Users cannot log in through the provider until it is enabled.
+    
+    AuthProvider:
+      anyOf:
+        - $ref: "#/components/schemas/AnonymousAuthProvider"
+        - $ref: "#/components/schemas/EmailPasswordAuthProvider"
+        - $ref: "#/components/schemas/ApiKeyAuthProvider"
+        - $ref: "#/components/schemas/CustomJwtAuthProvider"
+        - $ref: "#/components/schemas/CustomFunctionAuthProvider"
+        - $ref: "#/components/schemas/AppleAuthProvider"
+        - $ref: "#/components/schemas/GoogleAuthProvider"
+        - $ref: "#/components/schemas/FacebookAuthProvider"
```

Comment:
hey looks like this is different from what there is in the spec. not sure what's going on w that. this is how it shows in the spec: 
<img width="824" alt="Screen Shot 2022-11-21 at 3 18 13 PM" src="https://user-images.githubusercontent.com/90647379/203150558-905d301e-5c1f-4933-950f-e85e5ac96f65.png">

Reply:
it shows 'BaseOAuthProvider' and 2 instances of 'NonApppleOAuthProvider', whereas i assume this is supposed to be 'AppleAuthProvider', 'GoogleAuthProvider', and 'FacebookAuthProvider'

Reply:
Yeah it's a redoc bug - see my other comment https://github.com/mongodb/docs-app-services/pull/239#discussion_r1028468382

Reply:
based on looking at your code, seems like a bug w the openapi spec renderer.